scripts/jest/config.build-devtools.js

```diff
@@ -43,6 +43,9 @@ packages.forEach(name => {
   ] = `<rootDir>/build/node_modules/${name}/$1`;
 });
 
+// https://jestjs.io/docs/en/webpack#mocking-css-modules
+moduleNameMapper['\\.(css|less)$'] = 'identity-obj-proxy';
```

Comment:
There's currently exists a `cssMock.js` but it doesn't seem to be used. With `identity-obj-proxy` 
> Then all your className lookups on the styles object will be returned as-is (e.g., styles.foobar === 'foobar'). 

-- https://jestjs.io/docs/en/webpack#mocking-css-modules

Reply:
Was this causing a problem?

Reply:
In the new test I'm using HooksTree which imports some css file. Without mocking it, jest throws.

Reply:
Ah, I see. I don't think we should import a DevTools view in a test like that. I'll comment above.